mailtoslack.go

```diff
@@ -23,15 +23,23 @@ var (
 )
 
 func mailHandler(origin net.Addr, from string, to []string, data []byte) {
-	msg, _ := email.ParseMessage(bytes.NewReader(data))
+	msg, err := email.ParseMessage(bytes.NewReader(data))
+	if err != nil {
+		log.Printf("error parsing message: %s", err)
```

Comment:
FYI, this is (almost) the same as:
```go 
log.Println("error parsing message:", err)
```
the only difference is that the above will add a new line at the end